test/Makefile

```diff
@@ -65,3 +65,14 @@ clean:
 	@$(ECHO_CLEAN)
 	-$(QUIET) rm -rf $(TEST_ARTIFACTS)
 	-$(QUIET) rm -f .vagrant/*.box
+
+compile_bpf_tests:
+	$(QUIET) make -C ../bpf/tests/bpf_tests clean
+	$(QUIET) make -C ../bpf/tests/bpf_tests all
+	go test bpf_tests/*.go -c -o /tmp/bpf-tests
+
+run_bpf_tests: compile_bpf_tests
+	sudo /tmp/bpf-tests -bpf-test-path ../bpf/tests/bpf_tests
+
+run_bpf_tests_verbose: compile_bpf_tests
+	sudo /tmp/bpf-tests -bpf-test-path ../bpf/tests/bpf_tests -test.v
```

Comment:
FWIW we have a `V` variable in the Makefiles so you can do something like

```
BPF_TEST_FLAGS:=
ifneq ($(V),0)
    BPF_TEST_FLAGS += -test.v
endif
```

So then if someone explicitly requests no verbosity (`make -C test run_bpf_tests V=0`), it will be quiet. Otherwise it's loud.

This is an option, don't have to do it this way though.